src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -251,12 +251,23 @@ void GcsServer::InitGcsActorManager(const GcsInitData &gcs_init_data) {
     return std::make_shared<rpc::CoreWorkerClient>(address, client_call_manager_);
   };
 
+  std::function<bool(std::shared_ptr<GcsActor>)> release_resources =
+      [](std::shared_ptr<GcsActor> actor) { return true; };
+
   if (RayConfig::instance().gcs_actor_scheduling_enabled()) {
     RAY_CHECK(gcs_resource_manager_ && gcs_resource_scheduler_);
     scheduler = std::make_unique<GcsBasedActorScheduler>(
         main_service_, gcs_table_storage_->ActorTable(), *gcs_node_manager_, gcs_pub_sub_,
         gcs_resource_manager_, gcs_resource_scheduler_, schedule_failure_handler,
         schedule_success_handler, raylet_client_pool_, client_factory);
+    release_resources = [this](std::shared_ptr<GcsActor> actor) {
+      if (actor != nullptr && actor->GetActorWorkerAssignment() != nullptr) {
+        return gcs_resource_manager_->ReleaseResources(
+            actor->GetActorWorkerAssignment()->GetNodeID(),
+            actor->GetActorWorkerAssignment()->GetResources());
+      }
+      return false;
```

Comment:
why do we need to return a boolean here? The return is not used I think?

Reply:
Fixed.